integration-cli/docker_api_images_test.go

```diff
@@ -115,7 +115,14 @@ func (s *DockerSuite) TestAPIImagesHistory(c *check.C) {
 	c.Assert(err, checker.IsNil)
 
 	c.Assert(historydata, checker.Not(checker.HasLen), 0)
-	c.Assert(historydata[0].Tags[0], checker.Equals, "test-api-images-history:latest")
+	var found bool
+	for _, tag := range historydata[0].Tags {
```

Comment:
Can't this use a `checker.Contains` ?

Reply:
```
12:30:19 docker_api_images_test.go:118:
12:30:19     c.Assert(historydata[0].Tags, checker.Contains, "test-api-images-history:latest")
12:30:19 ... obtained []string = []string{"test-api-images-history:latest"}
12:30:19 ... substring string = "test-api-images-history:latest"
12:30:19 ... obtained value is not a string and has no .String().
```

I think `checker.Contains` only works with `string.Contains` :sweat: